src/Between.php

```diff
@@ -164,6 +164,10 @@ public function isValid($value)
     {
         $this->setValue($value);
 
+        if (is_null($this->getMin()) || is_null($this->getMax())) {
```

Comment:
don't use <code>is_null</code> for consistency, use <code>$this->getMin() === null</code> instead, see https://github.com/zendframework/zf2/pull/6129 and https://github.com/zendframework/zf2/pull/6643